packages/client/hmi-client/src/views/Model.vue

```diff
@@ -85,6 +87,13 @@ watch([model, graphElement], async () => {
 	await renderer?.setData(g);
 	await renderer?.render();
 });
+
+// FIXME: update after Dec 8 demo
+const router = useRouter();
+const goToSimulationPlanPage = () => {
+	// FIXME: can't use RouteName.SimulationRoute because of dependency cycle.
+	router.push({ name: 'simulation' });
```

Comment:
What do you mean?

Reply:
I'd like to replace this with
```suggestion
	router.push({ name: RouteName.SimulationRoute });
```
But doing so causes a linting error because `router/index.ts` imports `Model.vue` and `Model.vue` would then import `router/index.ts`.

I'm pretty sure the solution is as simple as extracting the `RouteName` enum to its own file and importing it into both `router/index.ts` and `Model.vue`, but I want to do that in a follow up PR since it'll update a bunch of imports across the codebase.

Reply:
I agree, good call.